mobile-app/api-client/LocalplayService.ts

```diff
@@ -0,0 +1,38 @@
+import { GameState, Coordinates } from "../utils";
+import { HEX_API_URL } from "@env"
+
+const LOCALPLAY_GAMES_ENDPOINT = "/localplay/games";
+
+export async function initNewGameState(size?: number): Promise<GameState> {
+  console.debug(`Call initNewGameState() with size=${size}`);
+  console.debug(`Full URL: ${HEX_API_URL + LOCALPLAY_GAMES_ENDPOINT}`);
+  const response = await fetch(HEX_API_URL + LOCALPLAY_GAMES_ENDPOINT, {
```

Comment:
we don't usually use `+`to concatenate strings. Prefer template literals with backticks.

```js
await fetch(`${HEX_API_URL}${LOCALPLAY_GAMES_ENDPOINT}`)
```